app/assets/stylesheets/components/layout/_mobile_menu.scss

```diff
@@ -0,0 +1,143 @@
+@import "../../variables";
+
+.header__primary-navigation {
+  display: none;
+
+  @media (min-width: 900px) {
+    display: block;
+  }
+}
+
+.header__secondary-navigation {
+  position: relative;
+}
+
+.header__mobile-navigation {
```

Comment:
🤔 I think the prefix `header__` wasn't the right choice, because we are styling the `navigation` that `header` contains. We use the word "navigation" for taxons and categories: [app/assets/stylesheets/components/navigation](https://github.com/solidusio/solidus-demo/tree/master/app/assets/stylesheets/components/navigation)

I'll change `header__` to `menu__` for now. Let me know if it does make sense for you too!